client/client_test.go

```diff
@@ -280,11 +280,13 @@ func testOCIExporter(t *testing.T, sb integration.Sandbox) {
 	defer os.RemoveAll(destDir)
 
 	out := filepath.Join(destDir, "out.tar")
+	target := "example.com/buildkit/testoci:latest"
```

Comment:
Sorry for going back and forth, but could you add a testcase for `target := "latest"`, as it seems the most expected value of `org.opencontainers.image.ref.name`. (`example.com/buildkit/testoci:latest` is also correct though)

Reply:
Is there a place I can read up for this value? I would expect "target=latest" to produce either error or `docker.io/library/latest:latest`

Reply:
For OCI, I'd suggest not normalizing the string. i.e. `target=latest` should set the annotation to `latest`, literally.

For OCI manifest within Docker-OCI-hybrid image, I'd suggest setting the "tag" portion of the Docker-normalized name.